### DISCUSSION TITLE
Jupyerlab extension bundled as CommonJS

### DISCUSSION THREAD

**Discussion by moiney** [perspective-dev/perspective#1452] [Q&A]
Hi,

We're working with Perspective to build interactive grid on Jupyerlab. On our side, once we're proud of one of our notebooks, we usually run it through Voilà afterwards.

PerspectiveWidgets used to work on Voilà up to v0.7.0, where the perspective-jupyterlab extension switched from umd bundling to commonjs2. This made it impossible to directly import the JavaScript (in the browser) for PerspectiveWidgets during Voilà runtime of a Perspective cell.

Is there a way to publish perspective-jupyterlab as umd/amd i/o CommonJS ?  RequireJS doesn't seem to support loading those bundling.

**Comment by sc1f** [comment]
This remains broken in the latest release and is something we are looking into - I've started an issue #1454 to track progress as well.

**Comment by moiney** [comment]
Works perfectly fine with 0.10.0, thanks a lot for this !
